pkg/storaged/client.js

```diff
@@ -995,17 +995,31 @@ function stratis3_start() {
                                                              clevis_info ? [true, clevis_info] : [false, ["", ""]]);
                 };
 
+                client.stratis_set_overprovisioning = (pool, flag) => {
+                    // DBusProxy is smart enough to allow
+                    // "pool.Overprovisioning = flag" to just work,
+                    // but we want to catch any error ourselves, and
+                    // we want to wait for the method call to
+                    // complete.
+                    return stratis.call(pool.path, "org.freedesktop.DBus.Properties", "Set",
+                                        ["org.storage.stratis3.pool." + stratis3_interface_revision,
+                                            "Overprovisioning",
+                                            cockpit.variant("b", flag)
+                                        ]);
+                };
+
                 client.stratis_list_keys = () => {
                     return client.stratis_manager.ListKeys();
                 };
 
-                client.stratis_create_filesystem = (pool, name) => {
-                    return pool.CreateFilesystems([[name, [false, ""]]]);
+                client.stratis_create_filesystem = (pool, name, size) => {
+                    return pool.CreateFilesystems([[name, size ? [true, size.toString()] : [false, ""]]]);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18867-20230823-070115-c85fe390-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)